### PR TITLE
Initial commit for help text.

### PR DESCRIPTION
**Pull request by danid123**
If the ACR is supposed to be reusable, maybe they have different contact people on the vendor side for different proposals.

Instead of "Name of individual..." maybe it could be "Contact name or team representing the vendor." I think that would make it more likely that the person filling out the form might put a team email in the email field. 

Instead of "Include an email for the vendor if they are open to feedback on the ACR (optional)." Maybe "Include a contact email for questions or feedback on the ACR (optional)" Same for the the author. 

From the procurement officer standpoint, how likely is it that they would contact both with questions? 

Typo under legal disclaimer: releaed => released

### CODE REVIEW

File: src/data/helpText.yaml [hunk 1]
```diff
@@ -1,29 +1,28 @@
 product:
-  name: ""
+  name: "Name of offering (ie. your product or service)."
```
Comment (dmundra) [inline src/data/helpText.yaml:2]: Nitpick here, should be either `ie` no period or `i.e.`

File: src/data/helpText.yaml [hunk 2]
```diff
@@ -1,29 +1,28 @@
...
-  description: ""
+  description: "Describe the general offering. This should not be specific to a contract or RFP."
 author:
-  name: ""
-  company_name: ""
-  address: ""
-  email: ""
-  phone: ""
-  website: ""
+  name: "Contact name or team representing author of the ACR (optional)."
```
Comment (dmundra) [inline src/data/helpText.yaml:6]: Do we need to add `(optional)`? We haven't worked on making required fields yet so by default everything is optional now. I would say let the form direct the optional required and not rely on the help text?

File: src/data/helpText.yaml [hunk 3]
```diff
@@ -1,29 +1,28 @@
...
+  website: "Website for the vendor or offering (optional)."
 report:
-  report_date: ""
-  notes: ""
-  evaluation_methods_used: ""
-  legal_disclaimer: ""
-  repository: ""
-  feedback: ""
-  license: ""
+  report_date: "Date last modified (YYYY/MM/DD)."
```
Comment (dmundra) [inline src/data/helpText.yaml:19]: The current is actually 'MM/DD/YYYY'. Should I change the format to match the help text?

**Review by dmundra — CHANGES_REQUESTED** [review]
Added some comments.

File: src/data/helpText.yaml (hunk 1 above)
Comment (mgifford) [reply src/data/helpText.yaml:2]: Ok. Such a silly word.

File: src/data/helpText.yaml (hunk 2 above)
Comment (mgifford) [reply src/data/helpText.yaml:6]: Ok..  Might make sense for the Legal doc as it is. Ultimately we don't want them filling that in.

File: src/data/helpText.yaml (hunk 3 above)
Comment (mgifford) [reply src/data/helpText.yaml:19]: I was just making one up, but that's obviously been done too many times..  I think we should settle on some standard.  
https://iso.mit.edu/americanisms/date-format-in-the-united-states/

I've changed it to DD/MM/YYYY (and we should be consistent).  I could also be convinced by an ISO argument https://www.iso.org/iso-8601-date-and-time-format.html

But would be good not to be country specific.

**Review by dmundra — APPROVED** [review]
Thanks @mgifford for making the changes.

**Pull request merged**